tests/LayerPlanTest.cpp

```diff
@@ -398,6 +398,24 @@ class AddTravelTest : public LayerPlanTest, public testing::WithParamInterface<s
         const Point destination(500000, 500000);
         return layer_plan.addTravel(destination);
     }
+
+    /*!
+     * Runs the test, adding a travel move to the layer plan with the specified input.
+     * \param unretract_before_last_travel_move Whether to unretract before the last travel move of the travel path
+     * \return The resulting g-code path.
+     */
+    GCodePath run(bool unretract_before_last_travel_move)
+    {
+        const Point destination(500000, 500000);
```

Comment:
What was the reasoning for the values? Also use `constexpr`

Reply:
Basically adding a long travel move.

Reply:
We can't make this variable `constexpr` without also adding a `constexpr` constructor. That can be done, of course, but is a modification to ClipperLib.

Reply:
Most certainly out of scope